test/mpi/dtpools/configure.ac

```diff
@@ -0,0 +1,77 @@
+#
+# (C) 2018 by Argonne National Laboratory.
+#     See COPYRIGHT in top-level directory.
+#
+#                                               -*- Autoconf -*-
+# Process this file with autoconf to produce a configure script.
+
+AC_PREREQ([2.67])
+AC_INIT([dtpools], [0.0], [discuss@mpich.org])
+AC_CONFIG_SRCDIR([include/dtpools.h])
+AC_CONFIG_MACRO_DIRS([confdb])
+
+AC_CONFIG_HEADER(dtpoolsconf.h)
+AH_TOP([/* -*- Mode: C; c-basic-offset:4 ; -*- */
+/*
+ *  (C) 2018 by Argonne National Laboratory.
+ *      See COPYRIGHT in top-level directory.
+ */
+#ifndef DTPOOLSCONF_H_INCLUDED
+#define DTPOOLSCONF_H_INCLUDED
+])
+
+AH_BOTTOM([
+#endif /* !defined(DTPOOLSCONF_H_INCLUDED) */
+])
+
+# Checks for programs.
+AC_PROG_CXX
+AC_PROG_AWK
+AC_PROG_CC
+AC_PROG_CPP
+AC_PROG_INSTALL
+AC_PROG_LN_S
+AC_PROG_MAKE_SET
+
+AM_INIT_AUTOMAKE([-Wall -Werror foreign])
+AM_SILENT_RULES([yes])
+AM_MAINTAINER_MODE([enable])
+AM_PROG_AR
+
+LT_PREREQ([2.2.6])
+LT_INIT
+
+AC_ARG_VAR([DTP_NUM_OBJS],
+           [used to limit the number of objects to test from each pool])
+
```

Comment:
Why do we define `DTP_NUM_OBJS` here ? It looks like it is just an optional environment variable set by the user.

Reply:
@minsii This is to tell users that this env variable exists and how to use it. I can remove it if you want